app/models/car.rb

```diff
@@ -0,0 +1,3 @@
+class Car < ApplicationRecord
+  has_many :reservations, dependent: :destroy
+end

```

Comment:
- [ ] Hello @Tobinchilongo, so I think here Car should `belongs_to :user`, user and car have relation :upside_down_face: 
- [ ] Also I think car have an attache `the image` so you can say `has_one_attached :car_image`.